packages/storybook-builder/src/generate-setup-addons-script.ts

```diff
@@ -2,19 +2,11 @@
 
 export async function generateSetupAddonsScript() {
   return `
-import { createChannel as createPostMessageChannel } from '@storybook/channel-postmessage';
-import { createChannel as createWebSocketChannel } from '@storybook/channel-websocket';
```

Comment:
did you remove these 2 deps? I assumed they moved stuff to core or import from it in their own builders, is that so? if that, I think we don't need to depend on these 2 packages directly anymore

Reply:
> did you remove these 2 deps? I assumed they moved stuff to core or reexport and then import from it in their own builders, is that so? if that, I think we don't need to depend on these 2 packages directly anymore

Reply:
one if those channels is not longer maintained, another one is a part of the core now